drivers/hmis/app/models/hmis/hud/custom_data_element.rb

```diff
@@ -30,12 +30,15 @@ class Hmis::Hud::CustomDataElement < Hmis::Hud::Base
     :value_json,
     :value_string,
     :value_text,
+    :value_file,
```

Comment:
I believe this should be `value_file_id` since the array contains the column names. Looking at the usage, we don't need to load the file every time-- for example when checking `equal_for_merge?` its sufficient to compare the file ids